package.json

```diff
@@ -12,17 +12,26 @@
     "build:prod": "wireit",
     "serve:dev": "wireit",
     "serve:prod": "wireit",
-    "watch:dev": "npm run build:dev && (npm run serve:dev & npm run build:dev --watch & npm run check --watch)",
-    "watch:prod": "npm run build:prod && (npm run serve:prod & npm run build:prod --watch & npm run check --watch)",
+    "watch:dev": "npm run build:dev && (npm run serve:dev > /dev/null 2>&1 & WIREIT_FAILURES=continue npm run build-and-check:dev --watch)",
+    "watch:prod": "npm run build:prod && (npm run serve:prod > /dev/null 2>&1 & WIREIT_FAILURES=continue npm run build-and-check:prod --watch)",
```

Comment:
Do we really need watch:prod? When do we think we'd use it?

Reply:
I don't see why not. I think there will end being potentially significant differences that having a fast debug cycle will be useful for, e.g. `prod` will likely use Eleventy SSR, but `dev` won't.